### PR TITLE
24 - I am adding FTP

### PR DESCRIPTION
**Pull request by Rico-Smit**
I am adding FTP

Related to #24

### CODE REVIEW

File: content/topics/protocols/index.md [hunk 1]
```diff
@@ -108,3 +108,12 @@ UDP stands for User Datagram Protocol. UDP is specifically chosen for time-sensi
...
 * [Cloudflare-UDP](https://www.cloudflare.com/learning/ddos/glossary/user-datagram-protocol-udp/)
+
+### FTP (File Transfer Protocol)
+
+This protocol is a group of rules that determine how computers transfer files from one system to another using the internet. FTP works by opening a connection between 2 parties. There are different communication channels between the parties for example: An connection is designated as a data channel, where the sharing happens once the server has granted you access. The FTP does lack security, it does not encrypt the sent data. There for FTPS of SFTP is used to safely transfer data.
+
+**References:**
+
+* [FTP-MDN](https://developer.mozilla.org/en-US/docs/Glossary/FTP)
+* [FTP-Hostinger] (https://www.hostinger.com/tutorials/what-is-ftp)
```
Comment (bpires) [inline content/topics/protocols/index.md:119]:
```suggestion
- [MDN Web Docs Glossary: FTP](https://developer.mozilla.org/en-US/docs/Glossary/FTP)

**See also:**

- [Hostinger Tutorials: What is FTP - FTP Explained for Beginners](https://www.hostinger.com/tutorials/what-is-ftp)
- Videos:
  - [CertBros: FTP Explained | File Transfer Protocol | Cisco CCNA 200-301](https://www.youtube.com/watch?v=L9aZpg0ip70)
```

**Review by bpires — CHANGES_REQUESTED** [review]
I suggested some changes in the text and in the way we organize references. If you agree, I think we are good to merge.

**Comment by schalkneethling** [comment]
@Rico-Smit Let me know once you have had a chance to look at @bpires' review comments. Thanks!

**Comment by Rico-Smit** [comment]
> @Rico-Smit Let me know once you have had a chance to look at @bpires' review comments. Thanks!

I have reviewed what i can from my moblie device, i will be working tonight if something is missing or needs doing

**Comment by bpires** [comment]
What happened @Rico-Smit? I think you accidentally closed the PR.

**Comment by Rico-Smit** [comment]
> What happened @Rico-Smit? I think you accidentally closed the PR.

It seems I deleted the branch by accident, I restored it now.

**Pull request merged**